### DISCUSSION TITLE
Destroy an instance of a dropdown list

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [twbs/bootstrap#38805] [Q&A]
Hello, how can I destroy/uninitialize a dropdown instance? I'm using the dispose method, but when the trigger is clicked, the list still toggles and adds the classes.
v5.3

**Comment by Awazleon** [comment]
Hello, did you try by using remove ?
```
<div class="dropdown" id="mydropdown">...</div>

dropdown = document.getElementById("mydropdown");
dropdown.remove();
```
This will completely remove it from the DOM.

**Reply by MyNameTony1000** [reply]
Good afternoon, I need to uninitialize the element in order to configure it later in a completely different way, that is, I need the element itself

**Reply by Awazleon** [reply]
Ok, I see. Dropdown element is just a list of `<lu>,` `<li>,`. You can assign an id to each `<li>.` In that case, it will be easy to remove the one you want.
This can be something as following:

```
<li id="toto">...</li>
<li id="tata">...</li>
...
<li id="titi">...</li>

const toRemove = document.getElementById("tata"); 
toRemove.parentNode.removeChild(toRemove);
```
You can dig around that.

**Reply by MyNameTony1000** [reply]
I understood that the matter is in incorrect translation, I need not to delete any element, but to uninitialize the drop-down element. As a destroy method in any slider, which removes / unbinds the carousel and everything connected with it from the DOM element, but the element itself remains and you can continue to work with it

**Reply by Awazleon** [reply]
Honestly, I don't really understand what you mean by "uninitialize the drop-down element". Sorry for that.
But in any case, I guess that is more Javascript relative rather than BS5 relative.

**Reply by MyNameTony1000** [reply]
Dropdowns - standard component bootstrap. Have you ever used slick slider or owl carousel? They have a destroy method that removes the carousel from an element. I wanted to find same method for bootstrap dropdown
